ui/models/job.js

```diff
@@ -116,39 +119,36 @@ export default class JobModel {
     const jobTerm = jobIds.length > 1 ? 'jobs' : 'job';
 
     try {
-      notify(`Attempting to retrigger ${jobTerm} via actions.json`, 'info');
-
-      /* eslint-disable no-await-in-loop */
-      for (const id of jobIds) {
-        const job = await JobModel.get(repoName, id);
-        const decisionTaskId = await getGeckoDecisionTaskId(
-          job.push_id,
-          repoName,
-        );
-        const results = await TaskclusterModel.load(decisionTaskId, job);
-        const retriggerTask = results.actions.find(
-          result => result.name === 'retrigger',
-        );
+      notify(`Attempting to retrigger/add ${jobTerm} via actions.json`, 'info');
+
+      const jobs = await JobModel.getList(repoName, { id__in: jobIds.join() });
+      const uniquePerPushJobs = groupBy(jobs, job => job.push_id);
+      for (const [key, value] of Object.entries(uniquePerPushJobs)) {
```

Comment:
just a nit...  we usually put a blank line between ``const`` declarations and functional code.  I've missed a few, but please add one, if you would.  :)